apps/omg/lib/omg/ethereum_event_listener.ex

```diff
@@ -145,4 +146,15 @@ defmodule OMG.EthereumEventListener do
     Application.fetch_env!(:omg, :ethereum_events_check_interval_ms)
     |> :timer.send_after(self(), :sync)
   end
+
+  defp publish_data([%{event_signature: event_signature} | _] = data) do
+    # String.split("DepositCreated(address,uint256,address,uint256)", "(")
+    [event_signature, _] = String.split(event_signature, "(")
+    :ok = OMG.Bus.direct_local_broadcast(event_signature, {:data, data})
```

Comment:
since this is a broadcast to a more generic event bus (i.e. not restricted to contract events), maybe naming the topic directly after the ethereum `event Name` might lead to confusion/conflicts. Instead of `DepositCreated` (which might clash with other events related to a deposit being created), something like `root_chain:DepositCreated`? (not sure what conventions/style this would have to follow though)

Also for a cleaner and more testable approach, putting code that retrieves the `OMG.Bus` topic from `event_signature` fits under `Preprocessor` responsibilities.

Reply:
Tracked as https://github.com/omisego/elixir-omg/issues/1187